app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  validates :content, presence: true
+  validates :user_id, presence: true
+end
```

Comment:
![Screen Shot 2020-11-02 at 11 16 47 PM](https://user-images.githubusercontent.com/48804258/97925322-7d878a00-1d61-11eb-8635-a4cfb04dccf0.png)

- [ ] As you can see in the screenshot above ☝️ your post model should have validations both for the body (content) and the title. Yours is currently missing the title.